panic: assignment to entry in nil map

Hi folks!

I use guard v0.2.0 and I try to integrate it with our LDAP server (I did everything according to the [official guide](https://appscode.com/products/guard/0.2.0/guides/authenticator/ldap/)), however when I try to obtain a token I've got the error:

```
$ guard get token -o ldap --ldap.auth-choice 0 --ldap.username=evgeny.shmarnev --ldap.password=foobar -v=3 --analytics=false
I0711 10:31:22.971536    9002 logs.go:19] FLAG: --alsologtostderr="false"
I0711 10:31:22.973452    9002 logs.go:19] FLAG: --analytics="false"
I0711 10:31:22.973482    9002 logs.go:19] FLAG: --help="false"
I0711 10:31:22.973499    9002 logs.go:19] FLAG: --ldap.auth-choice="0"
I0711 10:31:22.973513    9002 logs.go:19] FLAG: --ldap.disable-pa-fx-fast="true"
I0711 10:31:22.973527    9002 logs.go:19] FLAG: --ldap.krb5-config="/etc/krb5.conf"
I0711 10:31:22.973538    9002 logs.go:19] FLAG: --ldap.password="foobar"
I0711 10:31:22.973549    9002 logs.go:19] FLAG: --ldap.realm=""
I0711 10:31:22.973559    9002 logs.go:19] FLAG: --ldap.spn=""
I0711 10:31:22.973570    9002 logs.go:19] FLAG: --ldap.username="evgeny.shmarnev"
I0711 10:31:22.973586    9002 logs.go:19] FLAG: --log_backtrace_at=":0"
I0711 10:31:22.973599    9002 logs.go:19] FLAG: --log_dir=""
I0711 10:31:22.973612    9002 logs.go:19] FLAG: --logtostderr="false"
I0711 10:31:22.973623    9002 logs.go:19] FLAG: --organization="ldap"
I0711 10:31:22.973636    9002 logs.go:19] FLAG: --stderrthreshold="0"
I0711 10:31:22.973647    9002 logs.go:19] FLAG: --v="3"
I0711 10:31:22.973660    9002 logs.go:19] FLAG: --vmodule=""
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/appscode/guard/util/kubeconfig.AddAuthInfo(0x7fffaea93203, 0xf, 0xc42063c000, 0x6e61443530333237, 0x6c6969)
	/home/tamal/go/src/github.com/appscode/guard/util/kubeconfig/utils.go:41 +0x10b
github.com/appscode/guard/auth/providers/ldap.(*TokenOptions).addAuthInfo(0xc4203170b0, 0xc420043bf0, 0x24, 0xc4203e4dc0, 0xc4205ec630)
	/home/tamal/go/src/github.com/appscode/guard/auth/providers/ldap/kubectl.go:148 +0x75
github.com/appscode/guard/auth/providers/ldap.(*TokenOptions).IssueToken(0xc4203170b0, 0x4, 0x7fffaea931d9)
	/home/tamal/go/src/github.com/appscode/guard/auth/providers/ldap/kubectl.go:76 +0x94
github.com/appscode/guard/commands.NewCmdGetToken.func1(0xc4205f2780, 0xc4200de700, 0x0, 0x8)
	/home/tamal/go/src/github.com/appscode/guard/commands/token.go:51 +0x152
github.com/appscode/guard/vendor/github.com/spf13/cobra.(*Command).execute(0xc4205f2780, 0xc4200de400, 0x8, 0x8, 0xc4205f2780, 0xc4200de400)
	/home/tamal/go/src/github.com/appscode/guard/vendor/github.com/spf13/cobra/command.go:766 +0x2c1
github.com/appscode/guard/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc4201b7400, 0xc4205f2f00, 0xc4205f2c80, 0xc4205a0570)
	/home/tamal/go/src/github.com/appscode/guard/vendor/github.com/spf13/cobra/command.go:852 +0x30a
github.com/appscode/guard/vendor/github.com/spf13/cobra.(*Command).Execute(0xc4201b7400, 0x1841c00, 0xc42004c118)
	/home/tamal/go/src/github.com/appscode/guard/vendor/github.com/spf13/cobra/command.go:800 +0x2b
main.main()
	/home/tamal/go/src/github.com/appscode/guard/main.go:14 +0x48
```

My setup:
 - kubernetes v1.9.7 (deployed via kubeadm)
 - guard v0.2.0 - deployed inside of kubernetes cluster
```
$ kubectl get po -n kube-system | grep guard
guard-5f678c4ddd-xh9jk                                          1/1       Running   0          57m
```

If you need any additional info, feel free to ask me.

--- a/util/kubeconfig/utils.go
+++ b/util/kubeconfig/utils.go
@@ -35,6 +35,7 @@ func AddAuthInfo(username string, authInfo *clientcmdapi.AuthInfo) error {
 			Preferences: clientcmdapi.Preferences{
 				Colors: true,
 			},
+			AuthInfos: map[string]*clientcmdapi.AuthInfo{},
 		}
 	}
 

